internal/testutil/unit/http_mocker.go

```diff
@@ -38,6 +45,10 @@ func IsReplay() bool {
 	return slices.Contains([]string{"yes", "1", "true"}, strings.ToLower(os.Getenv(EnvNameHTTPMockerReplay)))
 }
 
+func IsDataUpdate() bool {
+	return slices.Contains([]string{"yes", "1", "true"}, strings.ToLower(os.Getenv(EnvNameHTTPMockerDataUpdate)))
```

Comment:
consider using strconv.ParseBool, like [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/92cd53a42be98e785e2edc8f584fce0d99a38587/internal/config/advanced_cluster_v2_schema.go#L12) to have consitency with bool env. vars, 

it accepts: case "1", "t", "T", "true", "TRUE", "True",    not "yes" but it should be fine

Reply:
Thanks, didn't know about it. Changed in 369ceb09bb52cd29de415a6aded502c9edde9f97
Although it is a lot more verbose :/